README.md

```diff
@@ -124,10 +124,54 @@ mpirun --mca btl_vader_single_copy_mechanism none -n 6 python3 -m pace.driver.ru
 This repository was first developed at [AI2](https://github.com/ai2cm/pace) and the institute conserves an archived copy with the latest state before the NOAA took over.
 
 [contributors-shield]: https://img.shields.io/github/contributors/ai2cm/pace.svg
-[contributors-url]: https://github.com/NOAA-GFDL/pace/graphs/contributors
-[stars-shield]: https://img.shields.io/github/stars/NOAA-GFDL/pace.svg
-[stars-url]: https://github.com/NOAA-GFDL/pace/stargazers
-[issues-shield]: https://img.shields.io/github/issues/NOAA-GFDL/pace.svg
-[issues-url]: https://github.com/NOAA-GFDL/pace/issues
-[license-shield]: https://img.shields.io/github/license/NOAA-GFDL/pace.svg
-[license-url]: https://github.com/NOAA-GFDL/pace/blob/main/LICENSE.md
+[contributors-url]: https://github.com/ai2cm/pace/graphs/contributors
+[stars-shield]: https://img.shields.io/github/stars/ai2cm/pace.svg
+[stars-url]: https://github.com/ai2cm/pace/stargazers
+[issues-shield]: https://img.shields.io/github/issues/ai2cm/pace.svg
+[issues-url]: https://github.com/ai2cm/pace/issues
+[license-shield]: https://img.shields.io/github/license/ai2cm/pace.svg
+[license-url]: https://github.com/ai2cm/pace/blob/main/LICENSE.md
+
+## Running pace in containers
+A Docker image exists in the Github Container Registry associated with the NOAA-GFDL organization.
+This image is publicly accessible and can be used to run a Docker container to work with pace.
+The following are directions on how to setup the pace conda environment interactively in a container.
+
+The latest image can be pulled with the Docker as shown below or
+with any other container management tools:
+
+```shell
+docker pull ghcr.io/noaa-gfdl/pace_mpich:3.8
+```
+for MPICH installation of MPI; and
+```shell
+docker pull ghcr.io/noaa-gfdl/pace_openmpi:3.8
+```
+for OpenMPI installation of MPI.
+
+If permission issues arise during the pull, a Github personal token
+may be required.  The steps to create a personal token is found
+[here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+
+Once the token has been generated, the image can be pulled for example with with:
+```shell
+docker login --username GITHUB_USERNAME --password TOKEN
+docker pull ghcr.io/noaa-gfdl/pace_mpich:3.8
+```
+
+Any container management tools compatible with Docker images can be used
+to run the container interactively from the pulled image.
+With Docker, the following command runs the container interactively.
+```shell
+docker run -it pace_mpich:3.8
+```
+
+In the container, the `pace` conda environment using python version 3.8 is already activated.
```

Comment:
We shall move off python 3.8. The new sanctioned python is 3.11.7. Can be logged and done afterward to no block refactor

Reply:
@FlorianDeconinck horrible of me, this is incorrect.  The testing/original version of the Docker containers had the pace conda environment all set up, but the current versions do not.  The users have to go through the steps themselves or execute a script in the container to set up the conda environment.   This sentence has been fixed with no reference to the python version being used.  That could be added if wished :)

Reply:
But the script setups `python3.8` correct?

Reply:
My overall advice would be to deliver a fully installed system via the image and in that case make it a python 3.11.x. Or tech stack is fragile to the version of Python because we use the language AST capacities themselves and those are heavily tied to the version of Python used.